nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="bg-black-core text-white text-center h-[550px] lg:h-[750px]">
```

Comment:
Need to fix this as well
![Screenshot from 2023-11-09 17-30-55](https://github.com/canopas/canopas-website/assets/123158106/24890853-8d81-42ec-8df5-bef7922f761f)